.github/workflows/main.yml

```diff
@@ -53,7 +55,11 @@ jobs:
       - name: Build 3c
         run: |
           cd ${{env.builddir}}
-          cmake -G Ninja -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_RTTI=ON -DCMAKE_BUILD_TYPE="Debug" -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra" ${{github.workspace}}/depsfolder/checkedc-clang/llvm
+          cmake -G Ninja\
+            -DLLVM_TARGETS_TO_BUILD=X86\
+            -DCMAKE_BUILD_TYPE="Debug"\
+            -DLLVM_ENABLE_PROJECTS="clang"\
```

Comment:
I prefer a space before the line continuation character: it makes it easier to see that the character isn't part of the previous argument and makes `git --word-diff` work better.  Please see the `format_and_cleanup.matt` branch.  (I hope I didn't break anything else there; I can run the workflow if you want.)